### DISCUSSION TITLE
No way to fetch assets with a tag using the API

### DISCUSSION THREAD

**Discussion by dlang123** [immich-app/immich#14555] [Feature Request]
### The bug

There is no endpoint to get all assets associated with a tag in the API, and search Metadata doesn't allow one to search by tags.

### The OS that Immich Server is running on

Unraid

### Version of Immich Server

v1.21.0

### Version of Immich Mobile App

NA

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
na
```

### Your .env content

```Shell
na
```

### Reproduction steps

1. Use Immich API to try to get assets associated with a tag, and there isn't an endpoint.
2. Albums have a getAlbumInfo that returns a list of assets associated with that album.
3. Tags  have no way to use the API to get all the assets, and a tag isn't in the search metadata part.

**Comment by alextran1502** [comment]
Converting this to feature request

**Comment by cauenapier** [comment]
Yes, please. THis would be useful. I spent a couple of hours trying to figure this out.
Still trying to find an workaround to select all photos with a given tag.

**Comment by bo0tzz** [comment]
#15395
